src/main/java/com/spotify/reaper/cassandra/JmxProxy.java

```diff
@@ -133,13 +135,19 @@ static JmxProxy connect(Optional<RepairStatusHandler> handler, String host, Stri
    * @param port     port number to use for JMX connection
    * @param username username to use for JMX authentication
    * @param password password to use for JMX authentication
+   * @param addressTranslator if EC2MultiRegionAddressTranslator isn't null it will be used to translate addresses
    */
   static JmxProxy connect(Optional<RepairStatusHandler> handler, String host, int port,
-      String username, String password)
+      String username, String password, final EC2MultiRegionAddressTranslator addressTranslator)
       throws ReaperException {
     ObjectName ssMbeanName;
     ObjectName cmMbeanName;
     JMXServiceURL jmxUrl;
+
+    if(addressTranslator != null) {
+      host = addressTranslator.translate(new InetSocketAddress(host, port)).getHostString();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Introduce a new variable instead of reusing the parameter "host". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1226)